proposals/gc/Charter.md

```diff
@@ -0,0 +1,60 @@
+# WebAssembly Garbage Collection Subgroup Charter
+
+The Garbage Collection Subgroup is a sub-organization of the
+[WebAssembly Community Group](https://www.w3.org/community/webassembly/)
+of the W3C.
+As such, it is intended that its charter align with that of the CG. In particular, 
+the sections of the [CG charter](https://webassembly.github.io/cg-charter/) relating to
+[Community and Business Group Process](https://webassembly.github.io/cg-charter/#process),
+[Contribution Mechanics](https://webassembly.github.io/cg-charter/#contrib),
+[Transparency](https://webassembly.github.io/cg-charter/#transparency),
+and
+[Decision Process](https://webassembly.github.io/cg-charter/#decision)
+also apply to the Subgroup.
+
+## Goals
+
+The mission of this subgroup is to provide a forum for collaboration on the standardisation of garbage collection support for WebAssembly.
+
+## Scope
+
+The Subgroup will consider topics related to garbage collection for Wasm, including:
+
+- an instruction set for defining and manipulating managed data types,
+- type system rules for validating such instructions,
+- APIs for accessing managed data types outside Wasm or a Wasm engine,
+- tool and language assists for user-space GC in linear memory,
+- code generation for compilers targetting Wasm with GC extensions.
+
+## Deliverables
+
+### Specifications
+
+The Subgroup may produce several kinds of specification-related work output:
+
+- new specifications in standards bodies or working groups
+  (e.g. W3C WebAssembly WG or Ecma TC39),
+
+- new specifications outside of standards bodies
+  (e.g. similar to the LLVM object file format documentation in Wasm tool conventions).
+
+### Non-normative reports
+
+The Subgroup may produce non-normative material such as requirements
+documents, recommendations, and use cases.
```

Comment:
```suggestion
documents, recommendations, and case studies.
```

Just want to emphasize a larger-scale analysis of how languages will compile to WebAssembly. Use cases tend to focus on features one at a time, whereas case studies further address the interaction of features.